### DISCUSSION TITLE
A request

### DISCUSSION THREAD

**Discussion by TeGuy** [community/community#58871] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Can you Make "Diggus Maximus" mod forge version?
I have been playing Minecraft Java Modded for years but Diggus Maximus is the best Mod i ever seen but im playing on Forge So can you add a forge version?☺️

**Comment by queenofcorgis** [comment]
Hi @TeGuy ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Minecraft or Forge. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
